### PR TITLE
Add createThread threadFields documentation

### PR DESCRIPTION
**Pull request by jordrake**

### CODE REVIEW

File: _snippets/typescript-sdk/create-thread.mdx [hunk 1]
```diff
@@ -25,6 +25,14 @@ const res = await client.createThread({
...
   // their ID.
   labelTypeIds: ['lt_01HB924PME9C0YWKW1N4AK3BZA'],
+  
+  // Thread fields are created in settings where you can also copy
+  // their key.
+  threadFields: [{
+    key: 'my_string_field',
+    type: ThreadFieldSchemaType.String,
+    stringValue: 'any value',
+  }],
```
Comment (preslavmihaylov) [inline _snippets/typescript-sdk/create-thread.mdx:35]: could use one example for boolean as well or even better for enum as well (which also uses stringValue)

File: _snippets/graphql/create-thread.mdx [hunk 2]
```diff
@@ -44,7 +51,12 @@ mutation createThread($input: CreateThreadInput!) {
         }
       }
     ],
-    "labelTypeIds": ["lt_01HB924PME9C0YWKW1N4AK3BZA"]
+    "labelTypeIds": ["lt_01HB924PME9C0YWKW1N4AK3BZA"],
+    "threadFields": [{
+      "key": "my_string_field",
+      "type": "ThreadFieldSchemaType.String",
+      "stringValue": "any value",
+    }],
```
Comment (preslavmihaylov) [inline _snippets/graphql/create-thread.mdx:59]: same here (for more examples)

File: _snippets/graphql/create-thread.mdx [hunk 3]
```diff
@@ -44,7 +51,12 @@ mutation createThread($input: CreateThreadInput!) {
         }
       }
     ],
-    "labelTypeIds": ["lt_01HB924PME9C0YWKW1N4AK3BZA"]
+    "labelTypeIds": ["lt_01HB924PME9C0YWKW1N4AK3BZA"],
+    "threadFields": [{
+      "key": "my_string_field",
+      "type": "ThreadFieldSchemaType.String",
```
Comment (preslavmihaylov) [inline _snippets/graphql/create-thread.mdx:57]: this looks a bit dodgy, is this tested wrt how you specify an enum when passing in variables?

Comment (jordrake) [reply _snippets/graphql/create-thread.mdx:57]: Woops you're right, copy and paste error

File: _snippets/typescript-sdk/create-thread.mdx (hunk 1 above)
Comment (jordrake) [reply _snippets/typescript-sdk/create-thread.mdx:35]: Added some examples below

**Review by presmihaylov — APPROVED** [review]

**Pull request merged**